### PR TITLE
refactor: Rewrite version detection logic

### PR DESCRIPTION
**Pull request by sjinks**
* Use the same version detection code for all OSes;
* Do not look specifically for docker-compose-v1;
* Prefer `/usr/bin`

### CODE REVIEW

File: test/lando.spec.js [hunk 1]
```diff
@@ -45,15 +45,19 @@ describe('lando', () => {
   });
 
   describe('#bootstrap', () => {
-    it('should return a lando object with the default config', () => {
-      const lando = new Lando({logLevelConsole: 'warn'});
+    it('should return a lando object with the default config', function() {
+      // eslint-disable-next-line no-invalid-this
+      this.timeout(45000);
```
Comment (sjinks) [inline test/lando.spec.js:50]: Tests time out without this on Windows, and I cannot figure out why :-(

Comment (rinatkhaziev) [reply test/lando.spec.js:50]: the best type of comment :)

**Comment by rinatkhaziev** [comment]
I tested on Mac and native Windows (i.e. not WSL directly but via cmd.exe) - works fine.

File: test/lando.spec.js (hunk 1 above)
Comment (rinatkhaziev) [reply test/lando.spec.js:50]: during my testing on Windows I wasn't able to replicate this. There weren't any discernible delays.

**Review by rinatkhaziev — APPROVED** [review]

**Pull request merged**